### ISSUE TITLE
Lab 2 Error and Question Regarding Analysis Questions

### ISSUE THREAD

**Issue by lindaalvarez** [Watts-College/paf-515-template#5]
Hello @castower ,

I was just about finished with the lab and when I tried to knit the RMD file I got this error message: 

processing file: lab02_lindaalvarez.Rmd
Error in parse(text = code, keep.source = FALSE) : 
  <text>:1:47: unexpected INCOMPLETE_STRING
1: alist( warning=FALSE, message=FALSE, results= "hide' )
                                                  ^
Calls: <Anonymous> ... <Anonymous> -> handle_error -> eval -> parse_only -> parse
Invalid syntax for chunk options:

warning=FALSE, message=FALSE, results= "hide'

Please see documentation at https://yihui.org/knitr/options/.

Execution halted

I'm a little confused as to what I need to do to fix this.  Secondly, I have a question as to where you want us to put our answers to the analysis questions. Do you want us to add them to the bottom of the lab 2 RMD file?

**Comment by castower** [comment]
Hello @lindaalvarez,

It appears one of your code chunk's 'hide' settings has mismatched quotes.

You can format your questions the same as the example lab linked under Submission Instructions.

**Comment by lindaalvarez** [comment]
@castower For the deliverables it asks to create an RMD file. Does this mean we have to submit the updated file with our steps for lab 2 or should there be two different project data steps files?

**Comment by castower** [comment]
@lindaalvarez Yes, you'll create a new RMD and Markdown file for Lab 02 and just update your project_data_steps.R file (you’ll keep adding to this same file throughout the duration of the course).

**Comment by lindaalvarez** [comment]
@castower I have 2 final questions:

I have been able to answer the questions regarding the different themes, but I am getting an error for the overall evaluation. I think I have the term confused. For example I have
svi_2020_divisional <- svi_theme_variables(svi_2020_divisional)
svi_2020_divisional <- svi_theme_flags(svi_2020_divisional, .90)

svi_2020_divisional %>% filter(!is.na(F_THEME1)) %>% filter(E_TOTPOP_20 > 100) %>% arrange(desc(RPL_THEME1)) %>% head(10) %>% kbl() %>% kable_styling() %>% scroll_box(width = "100%")

and that runs perfectly, but when I try to do it overall: svi_2020_divisional %>% filter(!is.na(F_OVERALL)) %>% filter(E_TOTPOP_20 > 100) %>% arrange(desc(RPL_OVERALL)) %>% head(10) %>% kbl() %>% kable_styling() %>% scroll_box(width = "100%") 

I get an error. Do I need to define a variable for this? 

Second question,  in question 5 it asks for the most and least vulnerable tracts, is it asking for the FIPS_tract number? Or should I be looking for a term? I appreciate your time and help.

**Comment by castower** [comment]
Hi @lindaalvarez 
 1. The overall column names are RPL_THEMES and F_TOTAL
 2. Yes, you can give the FIPS tract number and describe what county the tracts belong to.

Let me know if I can provide any further clarity!

**Comment by lindaalvarez** [comment]
@castower While knitting my rmd file one final time I seem to have made an error at the end for question 5. 
# Calculate state SVI variables and flags for 2020
svi_2020_state <- svi_theme_variables(svi_2020_state)
svi_2020_state <- svi_theme_flags(svi_2020_state, .90)

# Find most vulnerable tracts in 2020 for THEME 1 - SES
svi_2020_state %>% filter(!is.na(F_THEME1)) %>% filter(E_TOTPOP_10 > 100) %>% arrange(desc(RPL_THEME1)) %>% head(10) %>% kbl() %>% kable_styling() %>% scroll_box(width = "100%")

This is the error I am getting: Error in `filter()`:
ℹ In argument: `E_TOTPOP_10 > 100`.
Caused by error:
! object 'E_TOTPOP_10' not found
Backtrace:
  1. ... %>% scroll_box(width = "100%")
  9. dplyr:::filter.data.frame(., E_TOTPOP_10 > 100)
 10. dplyr:::filter_rows(.data, dots, by)
 11. dplyr:::filter_eval(...)
 13. mask$eval_all_filter(dots, env_filter)
 14. dplyr (local) eval()
Error in filter(., E_TOTPOP_10 > 100) : 
Caused by error:
! object 'E_TOTPOP_10' not found

I'm not sure how to fix this

**Comment by castower** [comment]
@lindaalvarez 
You're filtering the 2020 data set so you'll need to update the column name to match the year

**Comment by lindaalvarez** [comment]
> @lindaalvarez You're filtering the 2020 data set so you'll need to update the column name to match the year

@castower I'm so sorry, I'm not grasping what has to be updated. I think my code is all showing for 2020, Where can I find the column name?

**Comment by castower** [comment]
@lindaalvarez 

Yes, sure!

E_TOTPOP_10 should be updated to E_TOTPOP_20

The column names have suffixes for the year.

**Comment by lindaalvarez** [comment]
> @lindaalvarez
> 
> Yes, sure!
> 
> E_TOTPOP_10 should be updated to E_TOTPOP_20
> 
> The column names have suffixes for the year.

I got it!! Thank you so much for your help
